stubs/auth/resources/lang/en/auth.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+return [
+     /*
+    |--------------------------------------------------------------------------
+    | Authentication Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used during authentication for various
+    | messages that we need to display to the user. You are free to modify
+    | these language lines according to your application's requirements.
+    |
+    */
+    'failed' => 'These credentials do not match our records.',
+    'throttle' => 'Too many login attempts. Please try again in :seconds seconds.',
+    'passwords' => [
+        'confirm' => [
+            'heading' => 'Confirm your password',
+            'subtitle' => 'Please continue your password before continuing',
+            'forgot_password_link' => 'Forgot your password?',
+            'confirm_password_action' => 'Confirm password'
+        ],
+        'email' => [
+            'heading' => 'Reset password',
+            'send_password_reset_link_action' => 'Send password reset link'
+        ],
+        'reset' => [
+            'heading' => 'Reset password',
+            'confirm_password_action' => 'Confirm Password',
+            'reset_password_action' => 'Reset Password'
+        ],
+    ],
+    'login' => [
+        'heading' => 'Sign in to your account',
+        'subtitle' => [
+            'pre' => 'Or',
+            'link' => 'create a new account',
+        ],
+        'remember_label' => 'Remember',
+        'forgot_password_link' => 'Forgot your password?',
+        'sign_in_action' => 'Sign in',
+    ],
+    'register' => [
+        'heading' => 'Create a new account',
+        'subtitle' => [
+            'pre' => 'Or',
+            'link' => 'sign in to your account',
+        ],
+        'confirm_password_label' => 'Confirm Password',
+        'register_action' => 'Register'
+    ],
+    'verify' => [
+        'heading' => 'Verify your email address',
+        'subtitle' => [
+            'pre' => 'Or',
+            'link' => 'sign out',
+        ],
+        'resent_verification_link' => 'A fresh verification link has been sent to your email address.',
+        'check_email_for_link' => 'Before proceeding, please check your email for a verification link.',
+        'email_not_received' => [
+            'pre' => 'If you did not receive the email, ',
```

Comment:
I get why this is done, but it doesn't necessarily work to have "pre" and "link" text separately like this. It works for what the current English text is, but that won't be the case for every language.

What I prefer doing for things like this is to use parameters where I can, which lets the translator move it about and put text before or after the link if they need to:

```
'email_not_received' => 'If you did not receive the email, :link.',
'email_not_received_link_text' => 'click here to request another',
```

That does make it a bit more awkward to handle how to actually put the anchor HTML in there though.

Thoughts?

Reply:
This doesn't feel right to me but I get your point.

Do you have any experience in doing this? I'm quite clueless on this right now.